JumpscaleCore/clients/ssh/SSHClientBase.py

```diff
@@ -429,11 +425,11 @@ def _execute_script(
             content += "\n"
 
         if die:
-            content = "set -ex\n{}".format(content)
+            content = "set -e\n{}".format(content)
```

Comment:
not interested in the debug output?

Reply:
Nope as it ruines the stdout returned by  the command